docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,223 @@
 .. _gammapy_1p1_release:
 
-1.0 (unreleased)
-----------------
+1.1 (May XXth 2023)
+-------------------
 
-- No changes yet
+- Released May XXth, 2023
+- 17 contributors
+- 128 pull requests since v1.0 (not all listed below)
+- 85 closed issues
+
+Summary
+~~~~~~~
+
+This release introduces a number of new features as well as some performance improvements.
+
+Support for energy dependent temporal models is introduced. Only template models are supported.
+They use a `RegionNDMap` object with an energy and a time axis. Note that these models
+are meant for event simulation cannot be used for modeling and fitting
+
+A tutorial demonstrating how to simulate events using this type of models has been added.
```

Comment:
The tutorial is still not complete.
Do we want to publish the one that is available now and then we update it in the next weeks?